### DISCUSSION TITLE
Assets-Tree: Exclude files from folder tree

### DISCUSSION THREAD

**Discussion by polymed** [pimcore/pimcore#8239] [Q&A / Support] [answered]
Hi Guys,

is it possible to exclude "Files" from assets tree?

Best regards

![image](https://user-images.githubusercontent.com/51081904/108696843-5caa9680-7502-11eb-8e2e-3f94cb8dc913.png)

**Comment by NiklasBr** [comment, marked as answer]
Check out [Custom Views](https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Class_Settings/Custom_Views.html) and the chapter on *Additional asset tree hiding items based on a property* where you can create advanced rules for which objects or assets to show in a folder.

**Reply by polymed** [reply]
Thank you very much
